packages/react-component-library/src/components/Button/constants.ts

```diff
@@ -15,4 +15,9 @@ const BUTTON_COLOR = {
   DANGER: 'danger',
 }
 
-export { BUTTON_SIZE, BUTTON_VARIANT, BUTTON_COLOR }
+const BUTTON_ICON_POSITION = {
```

Comment:
This needs to be:

```
const BUTTON_ICON_POSITION = {
  LEFT: 'left',
  RIGHT: 'right'
} as const
```

Otherwise you don't get an error in your IDE with arbitrary text.

![Screenshot 2020-04-02 at 09 55 52](https://user-images.githubusercontent.com/56078793/78229649-32a24d00-74c8-11ea-815d-5665b8f39f38.png)